app/views/projects/_form.html.erb

```diff
@@ -595,9 +595,12 @@ provided by the delivered project's software.
 
 </div>
       <br>
+      <div class="center">
       <% if ! is_disabled %>
-        <%= f.submit 'Submit', class:"btn btn-primary" %>
+        <%= f.submit 'Save (and continue)', name: 'continue', class:"btn btn-primary btn-submit" %>
+        <%= f.submit 'Submit', class:"btn btn-primary btn-submit" %>
```

Comment:
Love the functionality, but I fear users won't understand.  The button text needs to be extremely clear, e.g.:

* "Save (and continue editing)" or maybe "Save and Continue"
* "Submit (and exit editing)" or maybe "Submit and exit"

Something to make it really clear what the difference is.  I don't think big buttons are a problem on mobile - it should wrap,  and we *want* big buttons on mobile so people can use them.